paper-checkbox.html

```diff
@@ -110,15 +110,27 @@
       },
 
       attached: function() {
+        this._updateContents(false);
```

Comment:
This `false` is also the implicit default for the argument. Is it necessary to pass it here?